docs/source/frameworks/transformers.rst

```diff
@@ -1,38 +1,312 @@
+
+         service: "service.py:svc"
+         labels:
+         owner: bentoml-team
+         project: gallery
+         include:
+         - "*.py"
+         python:
+           packages:
+           - transformers
+           - torch
+
+   .. tab-item:: TensorFlow
+
+      .. code-block:: yaml
+          :caption: bentofile.yaml
+
+          service: "service.py:svc"
+          labels:
+          owner: bentoml-team
+          project: gallery
+          include:
+          - "*.py"
+          python:
+            packages:
+            - transformers
+            - tensorflow
+
+
+Fined-tuned Models
+------------------
+
+Fine-tuning pretrained models is a powerful practice that allows users to save computation cost and adapt state-of-the-art models to their 
+domain specific dataset. Transformers offers a variety of libraries for fine-tuning pretrained models. The example below fine-tunes a BERT 
+model with Yelp review dataset. To learn more, refer to the Transformers guide on 
+`fine-tuning pretrained models <https://huggingface.co/docs/transformers/main/en/training>`_.
+
+.. code-block:: python
+    :caption: train.py
+
+    from datasets import load_dataset
+    from transformers import AutoModelForMaskedLM, AutoTokenizer, Trainer, TrainingArguments
+
+    dataset = load_dataset("yelp_review_full")
+    tokenizer = AutoTokenizer.from_pretrained("bert-base-cased")
+
+    def tokenize_function(examples):
+        return tokenizer(examples["text"], padding="max_length", truncation=True)
+
+    tokenized_datasets = dataset.map(tokenize_function, batched=True)
+
+    small_train_dataset = tokenized_datasets["train"].shuffle(seed=42).select(range(1000))
+    small_eval_dataset = tokenized_datasets["test"].shuffle(seed=42).select(range(1000))
+
+    model = AutoModelForMaskedLM.from_pretrained("bert-base-cased", num_labels=5)
+
+    training_args = TrainingArguments(output_dir="test_trainer", evaluation_strategy="epoch")
+
+    trainer = Trainer(
+        model=model,
+        args=training_args,
+        train_dataset=small_train_dataset,
+        eval_dataset=small_eval_dataset,
+    )
+
+    trainer.train()
+
+Saving a Fine-tuned Model
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the model is fine-tuned, create a Transformers 
+`Pipeline <https://huggingface.co/docs/transformers/main/en/pipeline_tutorial>`_ with the model and save to the BentoML model 
+store. By design, only Pipelines can be saved with the BentoML Transformers framework APIs. Models, tokenizers, feature extractors, 
+and processors, need to be a part of the pipeline first before they can be saved. Transformers pipelines are callable objects therefore 
+the signatures of the model are saved as :code:`__call__` by default.
+
+.. code-block:: python
+    :caption: train.py
+
+    import bentoml
+    from transformers import pipeline
+
+    unmasker = pipeline('fill-mask', model=model, tokenizer=tokenizer)
+
+    bentoml.transformers.save_model(name="unmasker", pipeline=unmasker)
```

Comment:
Maybe also have a quick two lines saying 
```
To quickly load the pipeline instance back for testing and debugging purposes:
bentoml..transformers.load_model(options_here)
```

Reply:
Added a paragraph for load_model without code sample since this is not specific to the Transformers module.
```rtf
To load the model for testing and debugging, use :code:`bentoml.transformers.load_model` with the :code:`unmasker:latest` tag.
```

Reply:
lgtm.